disnake/permissions.py

```diff
@@ -571,6 +574,14 @@ def send_messages_in_threads(self) -> int:
         """
         return 1 << 38
 
+    @flag_value
+    def moderate_members(self) -> int:
+        """:class:`bool`: Returns ``True`` if a user can moderate members.
```

Comment:
```suggestion
        """:class:`bool`: Returns ``True`` if a user can perform limited moderation actions (time out).
```
Some more details, this is the current description in the UI.